docs/canvas/canvas-share-workpad.asciidoc

```diff
@@ -0,0 +1,34 @@
+[role="xpack"]
+[[workpad-share-options]]
+== Share your workpad
+
+When you are ready to share your workpad, create a PDF, or export your workpad.
+
+[float]
+[[create-workpad-pdf]]
+=== Create a PDF
+
+To view your workpad outside of Kibana, generate a PDF. 
+
+. If you are using a basic license, enable reporting in your `config/kibana.yml` file.
+
+. From your workpad, click the *Share workpad* icon in the upper left corner, then select *PDF reports*.
+
+. Click *Generate PDF*. 
++
+image::images/canvas-generate-pdf.gif[]
+
+[float]
+[[export-workpad]]
+=== Export your workpad
+
+To share your workpads with another author, export your workpads as JSON files.
+
+. Go to the *Canvas workpads* page. 
```

Comment:
It's not a page per se. Clicking on the Workpad button will display a modal that pops up on top of your active workpad. We should maybe add a screenshot of the button here.

Reply:
I'm actually talking about this page:

<img width="968" alt="Screen Shot 2019-08-21 at 1 32 46 PM" src="https://user-images.githubusercontent.com/38990715/63458310-350fd500-c418-11e9-8742-78aaf9481422.png">